staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_clientset.go

```diff
@@ -89,7 +89,7 @@ func (g *genClientset) GenerateType(c *generator.Context, t *types.Type, w io.Wr
 	sw.Do(clientsetTemplate, m)
 	for _, g := range allGroups {
 		sw.Do(clientsetInterfaceImplTemplate, g)
-		// don't generated the default method if generating internalversion clientset
```

Comment:
NAK.
In k8s internalversion is a package name. It's not a typo.

Reply:
Thanks for your correct! :D